### DISCUSSION TITLE
Downgrading Question

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#906] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Can my school find out I downgraded or am I safe for now? Pls I need to know

**Comment by AshtonDavies** [comment]
No.

**Comment by PenguinLord561** [comment]
yes-  they probably will find out if they have basic amounts of detection. but if your school is stupid with tech there's a chance you might get away with it. 
your choise but if you want to be safe- dont do it

**Reply by AshtonDavies** [reply]
They can check devices in the admin console, but your device won't stick out among the many other not updated devices, so it's fine.

**Comment by CGRmags** [comment]
@AshtonDavies @PenguinLord561 I'm getting mixed answers I need a answer because I don't want to get in trouble

**Reply by AshtonDavies** [reply, marked as answer]
Short answer: no.
Downgrading is not visible by your administrator. If anything, they'd assume there's a bug or something when that particular device is on an older version than most others.

Similar thing for unenrollment. It blends with the hundreds or so unloaned Chromebooks in your school/district.

**Reply by CGRmags** [reply]
@AshtonDavies Can you answer my other question in discussion? Thanks

**Reply by AshtonDavies** [reply]
> @AshtonDavies Can you answer my other question in discussion? Thanks

I have no SKIOVOX for v120 experience as of now, nor any managed Chromebook to test with. I have to borrow from people I know to test that exploit.

**Comment by S-PScripts** [comment]
> Short answer: no. Downgrading is not visible by your administrator. If anything, they'd assume there's a bug or something when that particular device is on an older version than most others.
> 
> Similar thing for unenrollment. It blends with the hundreds or so unloaned Chromebooks in your school/district.

my school forces you to update your chromebook if you don't update it in like a month

**Reply by AshtonDavies** [reply]
> > Short answer: no. Downgrading is not visible by your administrator. If anything, they'd assume there's a bug or something when that particular device is on an older version than most others.
> > 
> > Similar thing for unenrollment. It blends with the hundreds or so unloaned Chromebooks in your school/district.
> 
> my school forces you to update your chromebook if you don't update it in like a month

Oh yeah, forced updates are harder to avoid and probably can't be stopped with CAUB. The only method I know for bypassing administrator demands is with the Bypass DNS server, which no one really uses. 🫤

**Reply by FunnyCorgi** [reply]
@AshtonDavies Can you provide a way to set that bypass DNS up?
